tests/functional/h/views/admin/test_permissions.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+
+class TestAdminPermissions:
+    PAGES = (
+        # URL, accessible by role.Staff?
```

Comment:
This might make the comment slightly clearer:

```suggestion
        # (URL, accessible by role.Staff?)
```

Or ... use a `namedtuple` :)